tests/common/broadcom_data.py

```diff
@@ -1,5 +1,2 @@
-SWITCH_HWSKUS = ["Force10-S6000", "Accton-AS7712-32X", "Celestica-DX010-C32",
-                 "Seastone-DX010", "Celestica-E1031-T48S4"]
-
-def is_broadcom_device(hwsku):
-    return hwsku in SWITCH_HWSKUS
+def is_broadcom_device(dut):
+    return dut.get_asic_type() == "broadcom"
```

Comment:
I believe AnsibleHost has a field called asic_type

return dut['asic_type'] == 'broadcomm'

Reply:
https://github.com/Azure/sonic-mgmt/pull/1531/files#diff-388cd8aa68de5bbbd57b2aafe842dccfR140

tamer is correct. let's use that.

Reply:
Done! Thanks for pointing that out.